drivers/dac/ad3530r/ad3530r.c

```diff
@@ -46,12 +46,65 @@
 #define AD3530R_CRC_BUFF_LEN(x)             (x) ? 3 : 4
 #define AD3530R_CRC_INDEX(x)                AD3530R_ADDR_INDEX(x) + 2
 
+/**
+ * @enum reg_set_offset
+ * @brief AD353xr offset for register base addresses
+ */
+enum reg_set_offset {
```

Comment:
is this enum necessary ? should there be an extra entry in it for ad3531 ?

Reply:
Actually, this is necessary for ad3532r which will be released after this part which has an offset for different register sets. I've removed the entry for ad3532r for time being.

Shall I remove this enum as a whole and add it later for ad3532r release?